### DISCUSSION TITLE
Selectable random seed range

### DISCUSSION THREAD

**Discussion by devilkkw** [hlky/stable-diffusion-webui#624] [Feature suggestions]
I like your UI so much, is it possible to adding over seed number input a selectable range?
i mean for example select range from 0 to 1000, 0 to 10000, etc, or maybe 2 textbox where insert min and max value.and if these was blank, use current randomiza function.
thank you in advance.

**Comment by slashedstar** [comment]
What would be the use of this? It's not like the seeds have any particular characteristic, like, seeds 10 to 20 vs 1010 to 1020, or a small vs a large range, the outputs would be equally random, so I'm quite curious on this

**Comment by devilkkw** [comment]
after some try, the seed number change the result style of generated image, like more realistic or more cartoon style, so for me limit range are an option to obtain results i want.
